<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoggingWidget</class>
 <widget class="QWidget" name="LoggingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="connect_lbl">
     <property name="text">
      <string>Connect to the intra</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="username_lineEdit">
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>name.lastname@epitech.eu</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="password_lineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="show_password_checkbox">
     <property name="text">
      <string>Show password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="remember_user_checkbox">
     <property name="text">
      <string>Remember this username</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="remember_password_checkbox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Remember password</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="connect_pushbutton">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
